[2024-02-28T13:28:52.712+0530] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_weather_data_1.fetch_files manual__2024-02-27T05:16:26.532787+00:00 [queued]>
[2024-02-28T13:28:52.714+0530] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_weather_data_1.fetch_files manual__2024-02-27T05:16:26.532787+00:00 [queued]>
[2024-02-28T13:28:52.714+0530] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-02-28T13:28:52.717+0530] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): fetch_files> on 2024-02-27 05:16:26.532787+00:00
[2024-02-28T13:28:52.720+0530] {standard_task_runner.py:60} INFO - Started process 4836 to run task
[2024-02-28T13:28:52.722+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_weather_data_1', 'fetch_files', 'manual__2024-02-27T05:16:26.532787+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/Data_fetcher.py', '--cfg-path', '/var/folders/th/tnjyp6yd1cj78nysn5t4jl6h0000gn/T/tmp0qy5kbgz']
[2024-02-28T13:28:52.723+0530] {standard_task_runner.py:88} INFO - Job 84: Subtask fetch_files
[2024-02-28T13:28:52.735+0530] {task_command.py:423} INFO - Running <TaskInstance: fetch_weather_data_1.fetch_files manual__2024-02-27T05:16:26.532787+00:00 [running]> on host aniks-macbook-air.local
[2024-02-28T13:28:52.751+0530] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fetch_weather_data_1' AIRFLOW_CTX_TASK_ID='fetch_files' AIRFLOW_CTX_EXECUTION_DATE='2024-02-27T05:16:26.532787+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-27T05:16:26.532787+00:00'
[2024-02-28T13:29:20.102+0530] {python.py:202} INFO - Done. Returned value was: None
[2024-02-28T13:29:20.120+0530] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=fetch_weather_data_1, task_id=fetch_files, execution_date=20240227T051626, start_date=20240228T075852, end_date=20240228T075920
[2024-02-28T13:29:20.165+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-28T13:29:20.179+0530] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
